src/App.js

```diff
@@ -17,6 +20,7 @@ function App() {
     <>
       <Navigation />
       <Routes>
+        <Route path="*" element={<ErrorPage />} />
```

Comment:
suggestion : try creating a separate file for the routes for improving the code structure